protected/humhub/config/common.php

```diff
@@ -196,7 +196,7 @@
             ],
         ],
         'mutex' => [
-            'class' => \yii\mutex\MysqlMutex::class,
```

Comment:
@yurabakhtin Please keep this as it is. I'll change it on client side via local Config file.